src/target/riscv/riscv.c

```diff
@@ -5411,24 +5577,21 @@ static const struct command_registration riscv_exec_command_handlers[] = {
 		.handler = riscv_set_ebreakm,
 		.mode = COMMAND_ANY,
 		.usage = "[on|off]",
-		.help = "Control dcsr.ebreakm. When off, M-mode ebreak instructions "
-			"don't trap to OpenOCD. Defaults to on."
+		.help = "DEPRECATED! use '<target_name> configure/cget -ebreak'"
```

Comment:
If the help message is expanded like these, it will hopefully be more clear to users:

```suggestion
		.help = "DEPRECATED! use '<target_name> configure -ebreak' or '<target_name> cget -ebreak'"
```

Reply:
Addressed.